From aad5b76e0e2743fb9564c833c1f55d4f6ea65efe Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Thu, 2 Jul 2015 14:26:35 +0800
Subject: [PATCH] MLK-11123-19 ARM: dts: disable otg features for
 imx6ul-14x14-ddr3-arm2 board

Disable all otg features for otg1 port of imx6ul-14x14-ddr3-arm2 board, then
the otg port can only do role swtich by ID pin state.

Signed-off-by: Li Jun <jun.li@freescale.com>
---
 arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
index 7391689..b34c376 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
@@ -313,6 +313,9 @@
 	vbus-supply = <&reg_usb_otg1_vbus>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usb_otg1_id>;
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.8.0

